tests/test_packers.py

```diff
@@ -78,6 +78,24 @@ def testWithFilter(self):
         self.assertLessEqual(packedId.bit_length(), packer.maxBits)
         self.assertEqual(packer.unpack(packedId), dataId)
 
+    def test_from_config(self):
```

Comment:
There's a lot of `if:else` logic in the new `__init__` method above: I'd like to see that explored with some simple tests here, including that appropriate mis-configuration raises. py-cov should help a lot here.